.goreleaser.yml

```diff
@@ -7,6 +7,11 @@ builds:
       - linux
     goarch:
       - amd64
+      - arm64
+
+# Build one binary for Mac OS (arm64 and amd64)
+universal_binaries:
+  - replace: true
```

Comment:
From the looks of the build logs, it seems like this will make it so that we don't upload the `darwin_amd64` binary anymore.

This could break things for anyone who's expecting that to be in the releases.

I'm not sure if it's a big deal honestly 🤷‍♂️